.htaccess

```diff
@@ -119,6 +122,7 @@
 	RewriteRule ^(.*)/topics/authentication$ /$1/developer_guides/security/authentication [R=301,L]
 	RewriteRule ^(.*)/topics/caching$ /$1/developer_guides/performance/caching [R=301,L]
 	RewriteRule ^(.*)/topics/commandline$ /$1/developer_guides/cli [R=301,L]
+	RewriteRule ^/topics/datamodel/?$ /en/developer_guides/model/data_model_and_orm  [R=301,L]
```

Comment:
Maybe follow the established convention for now, we could review and do a bulk improvement later?

We could do a separate review of rewrites looking for dups later, what is most important is actually having less 404s :)